build.gradle

```diff
@@ -36,8 +36,8 @@ import org.opensearch.gradle.test.RestIntegTestTask
 buildscript {
     ext {
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
-        opensearch_version = System.getProperty("opensearch.version", "2.0.0-rc1-SNAPSHOT")
```

Comment:
Can we make corresponding changes in `.github/security.yml` file to move to 2.0 branch as well?